### PR TITLE
[fr] use `const` instead of `var` in code examples

### PR DESCRIPTION
**Pull request by cricrio**
### Description

I replaced all the 'var' by 'const' in the code examples.

### CODE REVIEW

File: files/fr/web/api/document/getelementbyid/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Si vous avez besoin d'accéder à un élément qui n'a pas d'ID, vous pouvez uti
 ## Syntaxe
 
 ```js
-var element = document.getElementById(id);
+const element = document.getElementById(id);
 ```
```
Comment (cw118) [inline files/fr/web/api/document/getelementbyid/index.md:16]: Let's change this to be the [same as the English page](https://github.com/mdn/content/blob/main/files/en-us/web/api/document/getelementbyid/index.md?plain=1#L20-L22) :)

````suggestion
```js-nolint
getElementById(id)
```
````

**Review by cw118 — DISMISSED** [review]
Thank you for this PR @cricrio ! Just one small suggestion :)

File: files/fr/web/api/document/getelementbyid/index.md (hunk 1 above)
Comment (cricrio) [reply files/fr/web/api/document/getelementbyid/index.md:16]: I made the change :)

**Review by tristantheb — COMMENTED** [review]
Content updated from [`mdn/content:Document.getElementById() • be8f7f1`](https://github.com/mdn/content/blob/main/files/en-us/web/api/document/getelementbyid/index.md?plain=1)

File: files/fr/web/api/document/getelementbyid/index.md [hunk 2]
```diff
@@ -1,102 +1,109 @@
...
 
 ### Valeur de retour
 
-Un objet {{domxref("Element")}} décrivant l'objet élément du DOM correspondant à l'ID spécifié ou `null` si aucun n'a été trouvé dans le document.
+Un objet [`Element`](/fr/docs/Web/API/Element) décrivant l'objet élément du DOM correspondant à l'identifiant (_ID_) spécifié ou `null` si aucun n'a été trouvé dans le document.
 
-## Exemple
+## Exemples
 
 ### Contenu HTML
```
Comment (cw118) [inline files/fr/web/api/document/getelementbyid/index.md:38]:
```suggestion
### HTML
```

File: files/fr/web/api/document/getelementbyid/index.md [hunk 3]
```diff
@@ -1,102 +1,109 @@
...
-    <button onclick="changeColor('blue');">blue</button>
-    <button onclick="changeColor('red');">red</button>
+    <p id="para">Un peu de texte ici</p>
+    <button onclick="changerCouleur('blue');">bleu</button>
+    <button onclick="changerCouleur('red');">rouge</button>
   </body>
 </html>
 ```
 
 ### Contenu JavaScript
```
Comment (cw118) [inline files/fr/web/api/document/getelementbyid/index.md:53]:
```suggestion
### JavaScript
```

File: files/fr/web/api/document/getelementbyid/index.md [hunk 4]
```diff
@@ -1,102 +1,109 @@
...
-Un objet {{domxref("Element")}} décrivant l'objet élément du DOM correspondant à l'ID spécifié ou `null` si aucun n'a été trouvé dans le document.
+Un objet [`Element`](/fr/docs/Web/API/Element) décrivant l'objet élément du DOM correspondant à l'identifiant (_ID_) spécifié ou `null` si aucun n'a été trouvé dans le document.
 
-## Exemple
+## Exemples
 
 ### Contenu HTML
 
 ```html
 <html>
```
Comment (cw118) [inline files/fr/web/api/document/getelementbyid/index.md:41]:
```suggestion
<html lang="fr">
```

File: files/fr/web/api/document/getelementbyid/index.md [hunk 5]
```diff
@@ -1,102 +1,109 @@
...
+      // TypeError inattendu : parentDOM.getElementById n'est pas une fonction
     </script>
   </body>
 </html>
 ```
 
 S'il n'y a pas d'élément avec l'identifiant (`id`) fourni, cette fonction retourne `null`. À noter que le paramètre `id` est sensible à la casse, ainsi `document.getElementById("Main")` retournera `null` au lieu de l'élément `<div id="main">` étant donné que «&nbsp;M&nbsp;» et «&nbsp;m&nbsp;» sont différents pour cette méthode.
 
-**Les éléments absents du document** ne sont pas cherchés par `getElementById()`. Quand vous créez un élément et y assignez un ID, vous devez insérer l'élément dans l'arbre du document avec {{domxref("Node.insertBefore()")}} ou une méthode similaire avant de pouvoir y accéder avec `getElementById()` :
+Les éléments absents du document ne sont pas cherchés par `getElementById()`. Quand vous créez un élément et y assignez un ID, vous devez insérer l'élément dans l'arbre du document avec {{domxref("Node.insertBefore()")}} ou une méthode similaire avant de pouvoir y accéder avec `getElementById()`&nbsp;:
```
Comment (cw118) [inline files/fr/web/api/document/getelementbyid/index.md:98]:
```suggestion
Les éléments absents du document ne sont pas cherchés par `getElementById()`. Quand vous créez un élément et y assignez un ID, vous devez insérer l'élément dans l'arbre du document avec [`Node.insertBefore()`](/fr/docs/Web/API/Node/insertBefore) ou une méthode similaire avant de pouvoir y accéder avec `getElementById()`&nbsp;:
```

**Review by cw118 — COMMENTED** [review]
Quelques suggestions, merci déjà @tristantheb !

**Pull request merged**